ocrd/ocrd/processor/helpers.py

```diff
@@ -85,7 +87,12 @@ def run_processor(
     log.debug("Processor instance %s (%s doing %s)", processor, name, otherrole)
     t0_wall = perf_counter()
     t0_cpu = process_time()
-    processor.process()
+    mem_usage = memory_usage(proc=processor.process, interval=.1, timeout=None, timestamps=True, multiprocess=True, include_children=True)
```

Comment:
This causes a strange test failure in test_decorators, running the `processor.process` method 7 times instead of once.

```suggestion
    mem_usage = memory_usage(proc=processor.process, interval=.1, timeout=None, timestamps=True, multiprocess=True, include_children=True, max_iterations=1)
```

Seems to fix it but I am not sure I 100% understand how the function works.